gui/public/i18n/en/translation.ftl

```diff
@@ -19,6 +19,7 @@ tips-find_tracker = Not sure which tracker is which? Shake a tracker and it will
 tips-do_not_move_heels = Ensure your heels do not move during recording!
 tips-file_select = Drag & drop files to use, or <u>browse</u>.
 tips-tap_setup = You can slowly tap 2 times your tracker to choose it instead of selecting it from the menu.
+tips-turn_on_tracker = Remember to <b><em>turn on your tracker</em></b> before connecting it to the PC
```

Comment:
Shouldn't we suggest turning it on after? I've seen people have mysterious issues by turning it on before.
```suggestion
tips-turn_on_tracker = Remember to <b><em>turn on your tracker</em></b> after connecting it to the PC
```

Reply:
Nah it's dangerous to have DIY trackers on while plugged in at all, no difference for when it's turned on

Reply:
I changed the string, I think this can be resolved now?